src/autocomplete/stories/index.stories.js

```diff
@@ -130,6 +140,42 @@ storiesOf('autocomplete', module).add('Autocomplete', () => {
           )}
         </Autocomplete>
       </Box>
+      {/* Autocomplete with custom autocomplete item render */}
+      <Box padding={40}>
+        <Autocomplete
+          itemSize={50}
+          onChange={handleChange}
+          itemToString={item => (item ? item.name : '')}
+          items={itemsWithdetails}
+          renderItem={({ isHighlighted, isSelected, item, ...props }) => (
+            <Pane
+              display="flex"
+              alignItems="center"
+              justifyContent="left"
+              gap={10}
+              key={item.unique_id}
+              {...props}
+              paddingX={12}
+              cursor="pointer"
+            >
+              <Image src={item.image_src} width={50} />
+              <Text color="grey">{item.name}</Text>
+              <Small color="grey" size={300} flex={1}>
+                (Power: {item.power_score})
+              </Small>
+            </Pane>
+          )}
+        >
+          {({ getInputProps, getRef, getToggleButtonProps, inputValue, toggleMenu }) => (
+            <Box ref={ref => getRef(ref)} display="inline-block">
```

Comment:
What is the value of defining `ref` and passing it directly into the `getRef` function?

Have you considered changing it to be this?
```tsx
<Box ref={getRef} display="inline-block">
```